packages/tupaia-web/src/layout/Sidebar/DashboardMenu.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React, { useState } from 'react';
+import { useLocation, Link, useParams } from 'react-router-dom';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import { ButtonBase, Menu, MenuItem } from '@material-ui/core';
+import { Dashboard } from '@tupaia/types';
+import styled from 'styled-components';
+import { PANEL_GREY } from '../../constants';
+import { useDashboards } from '../../api/queries';
+import { DashboardCode } from '../../types';
+
+const MenuButton = styled(ButtonBase)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: ${PANEL_GREY};
```

Comment:
What do you think of putting this grey in the theme? I'm guessing that if we ever themed the app for a particular organisation then this colour may potentially be a part of that?

Reply:
yes that's a good idea. We will have to come up with a scheme to make all these theme colors make sense at some point. Perhaps it will need to evolve over time. I've added a panel section to the theme and used that through out the side bar for now.